angular.json

```diff
@@ -149,7 +154,12 @@
             "tsConfig": "projects/fullcalendar/tsconfig.lib.json",
             "project": "projects/fullcalendar/ng-package.json"
           }
-        },
+        ,          "configurations": {
```

Comment:
Q: I'm guessing this formatting issue is because of copy pasting, or from running the upgrade schematic?

Reply:
Yeah, it was a schematic.